aspnetcore/fundamentals/servers/index.md

```diff
@@ -20,56 +21,23 @@ An ASP.NET Core app runs with an in-process HTTP server implementation. The serv
 ASP.NET Core ships with the following:
 
 * [Kestrel server](xref:fundamentals/servers/kestrel) is the default, cross-platform HTTP server implementation.
-* IIS HTTP Server is an [in-process server](#in-process-hosting-model) for IIS.
+* IIS HTTP Server is an [in-process server](#hosting-models) for IIS.
 * [HTTP.sys server](xref:fundamentals/servers/httpsys) is a Windows-only HTTP server based on the [HTTP.sys kernel driver and HTTP Server API](/windows/desktop/Http/http-api-start-page).
 
 When using [IIS](/iis/get-started/introduction-to-iis/introduction-to-iis-architecture) or [IIS Express](/iis/extensions/introduction-to-iis-express/iis-express-overview), the app either runs:
 
-* In the same process as the IIS worker process (the [in-process hosting model](#in-process-hosting-model)) with the [IIS HTTP Server](#iis-http-server). *In-process* is the recommended configuration.
-* In a process separate from the IIS worker process (the [out-of-process hosting model](#out-of-process-hosting-model)) with the [Kestrel server](#kestrel).
+* In the same process as the IIS worker process (the [in-process hosting model](#hosting-models)) with the [IIS HTTP Server](#iis-http-server). *In-process* is the recommended configuration.
+* In a process separate from the IIS worker process (the [out-of-process hosting model](#hosting-models)) with the [Kestrel server](#kestrel).
 
 The [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module) is a native IIS module that handles native IIS requests between IIS and the in-process IIS HTTP Server or Kestrel. For more information, see <xref:host-and-deploy/aspnet-core-module>.
 
 ## Hosting models
```

Comment:
Reorder this to have kestrel first in the list of servers (as it's the only one that is cross plat).